chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -479,6 +479,7 @@ public TabTransformer getMainTabTransformer() {
 
   /**
    * Get the mutable list of context menu transformers in the main map view.
+   * The map view X and Y coordinates are supplied as a `Vector2` in the context menu's `mapviewPosition` property.
```

Comment:
Could you please clarify the unit for the coordinate system. Is it in pixels of the FX pane or in Minecraft blocks? Ideally I'd like to have access to both in different properties :)

Reply:
Looks like its in pixels of the FX pane. I updated the Javadocs and also added a property with the chunk coordinates (multiply by 16 to get blocks).